elks/arch/i86/drivers/char/crtc-6845.h

```diff
@@ -0,0 +1,55 @@
+#define OT_MDA 0
+#define OT_CGA 1
+#define OT_EGA 2
+#define OT_VGA 3
+
+#define N_DEVICETYPES 2
+
+#ifdef DEBUG
+static const char *type_string[] = {
+    "MDA",
+    "CGA",
+    "EGA",
+    "VGA",
+};
+#endif
+
+#if defined(CONFIG_CONSOLE_DUAL)
+struct hw_params {
+    unsigned short w;
+    unsigned short h;
+    unsigned short crtc_base;
+    unsigned vseg_base;
+    int vseg_bytes;
+    unsigned char init_bytes[16];
+    unsigned char n_init_bytes;
+    unsigned char max_pages;
+    unsigned short page_words;
```

Comment:
Can these two members move above init_bytes for clarity? Will be easier to read. n_init_bytes could also be after page_words and max_pages to make things even clearer, while still keeping the char members together.

Reply:
Done! Looks much nicer now.